string-manipulation-2/src/ransom-case.js

```diff
@@ -0,0 +1,12 @@
+/* exported ransomCase */
+function ransomCase(string) {
+  var String = '';
+  for (var i = 0; i < string.length; i++) {
+    if (i % 2 === 0) {
+      String += string[i].toLowerCase();
+    } else {
+      String += string[i].toUpperCase();
+    }
+  }
+  return String;
```

Comment:
Moving forward, let's not use capitalized variable name for now — that's a special naming convention for other stuff in Javascript